src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,23 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard/MovieCard';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  moviesFromServer: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ moviesFromServer = [] }) => (
+  <div className="movies">
+    <ul>
```

Comment:
```suggestion
  <ul className="movies">
```
цей клас потрібно вішати на сам контейнер (список) з картками. Зараз виглядає дивно. Картки не повинні бути в стовбчик
<img width="434" alt="image" src="https://user-images.githubusercontent.com/66570142/158365375-d53dba74-3246-4903-875a-e202a0912d12.png">

Reply:
Thanks!